services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/RateDetailsSummarySection.tsx

```diff
@@ -75,6 +109,12 @@ export const RateDetailsSummarySection = ({
                     )}
                 </SectionHeader>
 
+                <div aria-label="Rate ID" id="rateID" role="group">
```

Comment:
A couple things to check here
- What is the intended meaning here of the rate id? Is it information meant to be a [heading](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) or not? 
- IF it is, the only permitted aria roles for that element from the linked guide are a "heading", [tab](https://w3c.github.io/aria/#tab), [presentation](https://w3c.github.io/aria/#presentation) or [none](https://w3c.github.io/aria/#none). `role=definition` is not allowed.
- If you are using heading here mainly to get the heading-like styles, use CSS for that instead, not HTML

Reply:
I saw this mock up and a h4, though we wanted a h4 element. I'll change this to use a div with CSS.
<img src="https://user-images.githubusercontent.com/98117700/162227798-8128560c-381e-429a-b100-636c7cb630ce.png" width="300"/>

Reply:
just writing here what we discussed IRL for documentations - if a heading is being used it should be h3 since we want to only use headings in order (otherwise its an a11y fail). The Rate Details above is an h2 so if the rate name is a heading (I think we decided yes that makes sense, especially since we forsee the multi rate world coming) it should be an h3.  If the heading isnt clear via screenreader as read, an `aria-label` can be used.